client/cypress/fixtures/users.js

```diff
@@ -0,0 +1,7 @@
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
YOU SHOULD NOT DISABLE RULES IF THEY ARE SETUP IN YOUR REPOSITORY!
3 ways for not to get Eslint barking:
- follow rule (highly recommended);
- disable rule in Eslint config; 
- setup plugin which allows to use desired syntax (i'm not sure it is possible).

In here you need to export your object in a different way